uaclient/clouds/__init__.py

```diff
@@ -1,7 +1,7 @@
 import abc
 
 
-class UAPremiumCloudInstance(metaclass=abc.ABCMeta):
+class UAAutoAttachCloudInstance(metaclass=abc.ABCMeta):
```

Comment:
A nit, but I would prefer:

```suggestion
class AutoAttachCloudInstance(metaclass=abc.ABCMeta):
```

The `UA` prefix is redundant as the class is in a `uaclient` module, and `UAA` is an awkward-looking start to a symbol.

(If you do this, you'll obviously need to `sed` the other instances here rather than just accepting this suggestion, I'm just playing with the new functionality. :+1:)

Reply:
I'm +1 on fixing this now